Source/AppAuthTV/OIDTVServiceConfiguration.m

```diff
@@ -36,18 +35,16 @@ - (nullable instancetype)initWithCoder:(NSCoder *)aDecoder NS_DESIGNATED_INITIAL
 
 @end
 
-@implementation GTMTVServiceConfiguration
-
-@synthesize TVAuthorizationEndpoint = _TVAuthorizationEndpoint;
+@implementation OIDTVServiceConfiguration
 
 - (instancetype)init
     OID_UNAVAILABLE_USE_INITIALIZER(
-        @selector(initWithAuthorizationEndpoint:TVAuthorizationEndpoint:tokenEndpoint:));
+      @selector(initWithAuthorizationEndpoint:TVAuthorizationEndpoint:tokenEndpoint:))
 
 - (instancetype)initWithAuthorizationEndpoint:(NSURL *)authorizationEndpoint
-                                         tokenEndpoint:(NSURL *)tokenEndpoint
+                                tokenEndpoint:(NSURL *)tokenEndpoint
     OID_UNAVAILABLE_USE_INITIALIZER(
-        @selector(initWithAuthorizationEndpoint:TVAuthorizationEndpoint:tokenEndpoint:));
+      @selector(initWithAuthorizationEndpoint:TVAuthorizationEndpoint:tokenEndpoint:))
```

Comment:
Nit: Normal indentation is indeed 2 spaces, but when there's a line break due to the code not fitting on 1 line, the indentation is 4 spaces, so the original indentation was correct here and on line 42.

https://google.github.io/styleguide/objcguide.html#method-declarations-and-definitions
"If a method declaration does not fit on a single line, put each parameter on its own line. All lines except the first should be indented at least four spaces"